src/stories/containers/recognized-delegates/recognized-delegates.tsx

```diff
@@ -33,18 +63,60 @@ const RecognizedDelegatesContainer = () => {
       </ContainerInside>
       <Line />
       <ContainerInside>
-        <div>Last update</div>
-        <div>View the onchain transaction for recognized delegates</div>
-        <div>Tab Sections</div>
+        <ContainerPagerBar>
+          <PagerBar className="no-select" ref={null}>
+            <PagerBarLeft>
+              <StyledPagerBar
+                className="styledPagerBar"
+                label={currentMonth.toFormat('MMM yyyy').toUpperCase()}
+                // eslint-disable-next-line @typescript-eslint/no-empty-function
+                onPrev={() => {}}
+                // eslint-disable-next-line @typescript-eslint/no-empty-function
+                onNext={() => {}}
```

Comment:
Would be nice if we do this in a different way to avoid keep using the ignores here, maybe `() => null` can solve this

Reply:
ok, done